.claude/rules/prisma-db.md

```
@@ -34,7 +35,34 @@ globs:
+
+## Validate Constraints
+
+Prisma does not support `@@check` in `schema.prisma`. To add a validate constraint:
+
+1. Run `pnpm exec prisma migrate dev --create-only --name <description>` to generate the migration file without applying it
+2. Edit the generated `migration.sql` to add the validate constraint manually
+3. Run `pnpm exec prisma migrate dev` to apply
+
+After adding a validate constraint, add a comment to `docs/erd.md` under the relevant entity:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)